src/psycop_model_evaluation/binary/time/timedelta_plots.py

```diff
@@ -216,10 +220,18 @@ def plot_time_from_first_positive_to_event(
     x_labels = list(counts["time_from_first_positive_to_event_binned"])
     y_values = counts[0].to_list()
 
+    bin_unit2str = {
+        "h": "Hours",
+        "D": "Days",
+        "M": "Months",
+        "Q": "Quarters",
+        "Y": "Years",
+    }
```

Comment:
```suggestion
    bin_unit2str = {
        "h": "Hours",
        "D": "Days",
        "M": "Months",
        "Q": "Quarters",
        "Y": "Years",
    }
```

Excellent! We'll probably need these in other functions as well, so makes sense to extract them out of the function.